packages/forms/src/UIForm/UIForm.container.js

```diff
@@ -77,7 +77,7 @@ export default class UIForm extends React.Component {
 	constructor(props) {
 		super(props);
 		this.state = {
-			initialState: addErrorObject(this.props.data),
+			initialState: addErrorObject(this.props.data), // eslint-disable-line react/no-unused-state
```

Comment:
`initialState` is used, but outside of the class. Eslint parser doesn't detect that.